tests/py/casmutils/xtal/symmetry.py.in

```diff
@@ -29,6 +33,13 @@ class SymmetryGroupsTest(unittest.TestCase):
        sym_cube_lat=cu.xtal.symmetry.symmetrize(self.tet_lat, cu.xtal.symmetry.make_point_group(self.cubic_lat,self.tol)) 
        self.assertEqual(len(cu.xtal.symmetry.make_point_group(sym_cube_lat,self.tol)),48)
 
+    def test_structure_symmetrize(self):
```

Comment:
This seems like a separate test from `SymmetryGroupsTest`. You're testing symmetrization now, not whether point group and factor group generation works. It doesn't seem like you're using any of the members anyway. To avoid forever growing test classes, let's avoid stuffing existing ones with additional members unless it makes sense contextually. Just make a new derived `unittest.TestCase` (`SymmetrizationTest`?) and put your tests in there.